src/test/java/DukeTest.java

```diff
@@ -0,0 +1,20 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
```

Comment:
It might be easier to debug the code if there are more valid test methods in the test classes.